source/layouts/article.haml

```diff
@@ -15,3 +15,10 @@
 
       %p
         = link_to '&laquo; Zurück zur Übersicht', '/', class: 'back'
+
+      %p{:class => "share"}
+        Share on
+        = link_to 'Facebook', "https://www.facebook.com/sharer/sharer.php?u=#{config[:page_url]}#{current_page.url}"
+        = link_to 'Twitter', "https://twitter.com/intent/tweet?text=#{config[:page_url]}#{current_page.url}"
+        = link_to 'LinkedIn', "https://www.linkedin.com/cws/share?url=#{config[:page_url]}#{current_page.url}"
+        = link_to 'Email', "mailto:aFriend@example.com?&subject=#{config[:page_url]}#{current_page.url}"
```

Comment:
* I'd rather use the article's title as the email subject
* Better use `current_article.url`
* To be safe about the url concatenation use `File.join`
* I'd omit the sample email. The `link_to` (or `mail_to`) helper doesn't allow this, which is why I'd use a standard link

 `%a{ href: "mailto:?subject=#{current_article.title}&body=#{File.join(config[:page_url], current_page.url)}" } Email`